### PR TITLE
Added Health statements for Mashov website

### PR DESCRIPTION
**Pull request by UdiMizrachi**
Added Health statements for Mashov website
Changed environment variables - to select on which website the sign is needed, and also login information for Mashov website for each kid & school
Also fixed some Typo`s , and fixed the send image loop to send all *.png images in the /opt folder

### CODE REVIEW

File: dockerbot.py [hunk 1]
```diff
@@ -26,13 +48,25 @@ def handle(msg):
...
+
+            if v_SIGN_WEBSITE_MASHOV == 1:
+                if v_MASHOV_NUMBER_OF_KIDS > 0:
+                    for Mashov_Kid_Number in range(1, v_MASHOV_NUMBER_OF_KIDS, 1):
+                        bot.sendMessage(chat_id,"Starting Sign process at https://web.mashov.info/students/login for Kid Number -" + str(Mashov_Kid_Number))
+                        Prep_Switch_MASHOV_USER_DICT_ID_KID = v_MASHOV_USER_DICT_ID_KID[Mashov_Kid_Number]
+                        Prep_Switch_MASHOV_USER_DICT_ID_PWD = v_MASHOV_USER_DICT_ID_PWD[Mashov_Kid_Number]
+                        Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID = v_MASHOV_USER_DICT_ID_SCHOOL_ID[Mashov_Kid_Number]
+                        subprocess.check_output(['python', '/etc/Mashov_Health_Statements.py', '-u', Prep_Switch_MASHOV_USER_DICT_ID_KID, '-p', Prep_Switch_MASHOV_USER_DICT_ID_PWD , '-s', Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID, '-kn', str(Mashov_Kid_Number)])
+
```
Comment (AdamRussak) [inline dockerbot.py]: @UdiMizrachi  please seprate the education and the mashov in to 2 seperate commands:
example:
/sign_edu
/sign_mashov
feel free to set diffrent names.

File: dockerbot.py [hunk 2]
```diff
@@ -23,16 +45,85 @@ def handle(msg):
 
     logger.info(f"[{message_id}] Got command: {command}")
 
-    if command == '/sign':
+    if command == '/commands':
```
Comment (AdamRussak) [inline dockerbot.py]: change the commands to /? (much more user friendly) 
```suggestion
    if command == '/?':
```

File: dockerbot.py (hunk 1 above)
Comment (UdiMizrachi) [reply dockerbot.py]: Done,
Also added logic to the Mashov website to check if checkmarks already selected (At Mashov website you can remove/add checkmarks all day long, not as EDU website which is 1 run per day)

File: dockerbot.py [hunk 3]
```diff
@@ -4,10 +4,32 @@
 import subprocess, os, sys
 from telepot.loop import MessageLoop
 
+#Which Signing Website
+v_SIGN_PARENTS_EDUCATION_GOV_IL = int(os.getenv('SIGN_WEBSITE_EDUCATION_GOV_IL'))
+v_SIGN_WEBSITE_MASHOV = int(os.getenv('SIGN_WEBSITE_MASHOV'))
```
Comment (AdamRussak) [inline dockerbot.py:12]: why is it neededd? 
once you chose the command in telegram you get the script with the URL inside or is there another reason?

**Review by AdamRussak — CHANGES_REQUESTED** [review]
I'm missing the path for the kid's info (you removed it from docker-compose but didn't add it somewhere else)

File: dockerbot.py [hunk 4]
```diff
@@ -45,14 +45,14 @@ def handle(msg):
 
     logger.info(f"[{message_id}] Got command: {command}")
 
-    if command == '/commands':
+    if command == '/?':
```
Comment (bityob) [inline dockerbot.py:51]: don't need to. Use the botFather for that

**Pull request merged**